webapp/src/services/api.ts

```diff
@@ -1,6 +1,6 @@
 import { QueryReturnValue } from "@reduxjs/toolkit/dist/query/baseQueryTypes";
 import { createApi, fakeBaseQuery } from "@reduxjs/toolkit/query/react";
-import { AzimuthConfig, UtterancePatch } from "types/api";
+import { AzimuthConfig, HTTPExceptionModel, UtterancePatch } from "types/api";
```

Comment:
Don't forget to check the warnings in the terminal. This new import is not used:
```suggestion
import { AzimuthConfig, UtterancePatch } from "types/api";
```